packages/go_router/lib/src/state.dart

```diff
@@ -122,8 +122,7 @@ class GoRouterState {
 
   /// Get a location from route name and parameters.
   /// This is useful for redirecting to a named location.
-  @Deprecated(
-      'Uses GoRouter.of(context).routeInformationParser.namedLocation instead')
+  @Deprecated('Uses GoRouter.of(context).namedLocation instead')
```

Comment:
Should that be "Use" (rather than "Uses")?

Reply:
Done!